php_phongo.c

```diff
@@ -1062,16 +1111,21 @@ static bool php_phongo_apply_rc_options_to_uri(mongoc_uri_t *uri, bson_t *option
 		return true;
 	}
 
-	if (!bson_iter_init_find_case(&iter, options, "readconcernlevel")) {
+	if (!bson_iter_init_find_case(&iter, options, MONGOC_URI_READCONCERNLEVEL)) {
```

Comment:
This change is unrelated to the PR, but I noticed while fixing `bson_iter_init_find_case()` below that we weren't using the defined constant for this option. Let me know if you'd like it split out into a separate commit.